Makefile

```
@@ -258,22 +255,6 @@ validate-%: build ## validate specific check
 win: bundles ## cross build the binary for windows
 	$(BAKE_CMD) --set *.platform=windows/amd64 binary
 
-.PHONY: swagger-gen
-swagger-gen:
-	docker run --rm -v $(PWD):/go/src/github.com/docker/docker \
-		-w /go/src/github.com/docker/docker \
-		--entrypoint hack/generate-swagger-api.sh \
-		-e GOPATH=/go \
-		quay.io/goswagger/swagger:0.7.4
-
-.PHONY: swagger-docs
-swagger-docs: ## preview the API documentation
-	@echo "API docs preview will be running at http://localhost:$(SWAGGER_DOCS_PORT)"
-	@docker run --rm -v $(PWD)/api/swagger.yaml:/usr/share/nginx/html/swagger.yaml \
-		-e 'REDOC_OPTIONS=hide-hostname="true" lazy-rendering' \
-		-p $(SWAGGER_DOCS_PORT):80 \
-		bfirsh/redoc:1.14.0
-
```

Perhaps we should keep these, and forward them to the new makefile;

```suggestion
.PHONY: swagger-gen
swagger-gen:  ## generate swagger API types
	$(MAKE) -C api swagger-gen

.PHONY: swagger-docs
swagger-docs: ## preview the API documentation
	$(MAKE) -C api swagger-docs

```